ArraysHomework/ArraysHomework.cs

```diff
@@ -152,38 +200,46 @@ public void Lists()
 
             // BONUS reverse this list
             List<int> reverseMe = new List<int>() { 1, 2, 3, 4, 5 };
-
-            //Assert.IsTrue(reverseMe[0] == 5);
-            //Assert.IsTrue(reverseMe[1] == 4);
-            //Assert.IsTrue(reverseMe[2] == 3);
-            //Assert.IsTrue(reverseMe[3] == 2);
-            //Assert.IsTrue(reverseMe[4] == 1);
+            reverseMe.Reverse();
+            Assert.IsTrue(reverseMe[0] == 5);
+            Assert.IsTrue(reverseMe[1] == 4);
+            Assert.IsTrue(reverseMe[2] == 3);
+            Assert.IsTrue(reverseMe[3] == 2);
+            Assert.IsTrue(reverseMe[4] == 1);
 
             // ---------
-            
+
             // BONUS sort this list
             List<int> sortMe = new List<int>() { 14, 3, 33, 2, 12 };
-
-            //Assert.IsTrue(sortMe[0] == 2);
-            //Assert.IsTrue(sortMe[1] == 3);
-            //Assert.IsTrue(sortMe[2] == 12);
-            //Assert.IsTrue(sortMe[3] == 14);
-            //Assert.IsTrue(sortMe[4] == 33);
+            sortMe.Sort();
+            Assert.IsTrue(sortMe[0] == 2);
+            Assert.IsTrue(sortMe[1] == 3);
+            Assert.IsTrue(sortMe[2] == 12);
+            Assert.IsTrue(sortMe[3] == 14);
+            Assert.IsTrue(sortMe[4] == 33);
 
             // ----------
 
             // BONUS Convert sortMe to an array
+            int[] array = sortMe.ToArray();   
 
             // ----------
 
             // BONUS Check if sortMe Contains the value 2.
+            foreach(int item in sortMe)
```

Comment:
(This is just bonus)
How would you save this result into a variable, so e.g. `containsTwo` is `true` if the list contains 2 and `false` if it doesn't?
Also, although a loop is an absolutely valid approach, is there a simpler way of doing this check?